### PR TITLE
Feature about or contact page

### PR DESCRIPTION
**Pull request by LAEEQ18**
index page is the home one

about and contact page is for review

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Permission to use, copy, modify, and/or distribute this software for any
+purpose with or without fee is hereby granted.
+
+THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH
+REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY
+AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT,
+INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM
+LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR
+OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
+PERFORMANCE OF THIS SOFTWARE.
```
Comment (hanif-arif) [inline LICENSE:14]: remove license

File: gatsby-config.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    },
+
+    {
+      resolve: `gatsby-source-contentful`,
+      options: {
+        spaceId: `9jzhvd94c25e`,
+        // Learn about environment variables: https://gatsby.dev/env-vars
+        accessToken: process.env.ACCESS_TOKEN,
+      },
+    },
```
Comment (hanif-arif) [inline gatsby-config.js:44]: no need for contentful remove

File: package.json [hunk 3]
```diff
@@ -0,0 +1,61 @@
+{
+  "name": "intelsnap-website",
+  "private": true,
+  "description": "A static website for Intelsnap Inc.",
+  "version": "0.1.0",
+  "license": "0BSD",
```
Comment (hanif-arif) [inline package.json:6]: remove

File: package.json [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    "prettier": "^2.1.2"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/gatsbyjs/gatsby-starter-hello-world.git"
+  },
+  "bugs": {
+    "url": "https://github.com/gatsbyjs/gatsby/issues"
+  },
+  "homepage": "https://github.com/gatsbyjs/gatsby-starter-hello-world#readme",
```
Comment (hanif-arif) [inline package.json:57]: remove

File: package.json [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    "lint:fix": "eslint . --fix"
+  },
+  "dependencies": {
+    "@fortawesome/fontawesome-free": "^5.15.1",
+    "@fortawesome/fontawesome-svg-core": "^1.2.32",
+    "@fortawesome/free-brands-svg-icons": "^5.15.1",
+    "@fortawesome/free-solid-svg-icons": "^5.15.1",
+    "@fortawesome/react-fontawesome": "^0.1.12",
+    "babel-plugin-styled-components": "^1.11.1",
+    "bootstrap": "^4.5.3",
```
Comment (hanif-arif) [inline package.json:25]: remove bootstrap

File: package.json [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+    "gatsby": "^2.24.79",
+    "gatsby-image": "^2.4.21",
+    "gatsby-plugin-sharp": "^2.6.43",
+    "gatsby-plugin-styled-components": "^3.3.14",
+    "gatsby-source-contentful": "^3.0.6",
```
Comment (hanif-arif) [inline package.json:30]: do not need gatsby-source-contentful

File: src/components/Footer.css [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+  padding: 5px;
+  color: rgb(0, 0, 0);
+  font-size: 0.8rem;
+} */
+
+.Container-fluid {
+  background-color: #f8f6f5;
+  padding-left: 3px;
+  margin-left: auto;
+}
```
Comment (hanif-arif) [inline src/components/Footer.css:11]: Move this to Footer.js using styled components

Comment (hanif-arif) [reply src/components/Footer.css:11]: Remove separate css file

File: src/components/Footer.js [hunk 8]
```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import intel from "../images/intelsnap.png";
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import '../utils/font-awesome';
+import   "./Footer.css";
+import 'bootstrap/dist/css/bootstrap.min.css';
```
Comment (hanif-arif) [inline src/components/Footer.js:6]: Remove Footer.css and bootstrap

File: src/components/Navbar.css [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+  padding: 20px 10px;
+}
+
+@media screen and (max-width: 800px) {
+  .sticky {
+    flex-wrap: wrap;
+    padding: 15px;
+    flex-flow: wrap;
+  }
+}
```
Comment (hanif-arif) [inline src/components/Navbar.css:40]: Move to Navbar using styled components

File: src/components/Navbar.js [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+
+const Navbar = () => {
+    // const data = useStaticQuery(graphql `
+    //     query {
+    //         logo: file(relativePath: {eq: "logo.svg"}) {
+    //             publicURL
+    //         }
+    //     }
+
+    // `)
```
Comment (hanif-arif) [inline src/components/Navbar.js:19]: remove commented out code

File: src/components/Navbar.js [hunk 11]
```diff
@@ -0,0 +1,61 @@
+import { Link } from 'gatsby'
+import React from 'react';
+//import {Logowrapper} from "../components/logoelement";
+//import {useStaticQuery, graphql} from "gatsby";
+import logo from "../assets/logo.svg";
+import './Navbar.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import Nav from 'react-bootstrap/Nav';
```
Comment (hanif-arif) [inline src/components/Navbar.js:8]: Remove bootstrap, replace with Antd

File: src/components/Navbar.js [hunk 12]
```diff
@@ -0,0 +1,61 @@
+import { Link } from 'gatsby'
+import React from 'react';
+//import {Logowrapper} from "../components/logoelement";
+//import {useStaticQuery, graphql} from "gatsby";
```
Comment (hanif-arif) [inline src/components/Navbar.js:4]: remove commented out code

**Review by hanif-arif — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 1 above)
Comment (LAEEQ18) [reply LICENSE:14]: done

File: gatsby-config.js (hunk 2 above)
Comment (LAEEQ18) [reply gatsby-config.js:44]: done

File: package.json (hunk 5 above)
Comment (LAEEQ18) [reply package.json:25]: removed

**Comment by LAEEQ18** [comment]
done

**Pull request closed without merging**